contrib/platform/src/com/sun/jna/platform/win32/Kernel32.java

```diff
@@ -1413,6 +1413,71 @@ boolean CreateProcessW(String lpApplicationName, char[] lpCommandLine,
      */
     boolean GetVersionEx(OSVERSIONINFOEX lpVersionInfo);
 
+    /**
+     * The VerifyVersionInfoW function compares a set of operating system
+     * version requirements to the corresponding values for the currently
+     * running version of the system.This function is subject to manifest-based
+     * behavior.
+     * 
+     * @param lpVersionInformation
+     *            A pointer to an OSVERSIONINFOEX structure containing the
```

Comment:
it would be great if embedded code/code like sequences would be surrounded by {@code <data>} and if possible @link to the class. That is not required, but people might like it :-)